src/home/components/Connect.jsx

```diff
@@ -0,0 +1,232 @@
+
+  margin: 1rem;
+
+  &:hover,
+  &:active {
+    border: none;
+    outline: none;
+  }
+
+  ${state.endIndex >= Cards.length - 1 && "svg {transform: rotate(180deg);}"}
+`;
+
+const handleForward = () => {
+  if (state.endIndex <= Cards.length - 1) {
+    State.update({
+      startIndex: state.endIndex + 1,
+      endIndex: state.endIndex + 3,
+    });
+  } else {
+    State.update({
+      startIndex: 0,
+      endIndex: 2,
+    });
+  }
+};
+
+const CTA = styled.a`
+  color: #00ec97;
+  font-size: 24px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 120%; /* 28.8px */
+`;
+
+const Subheading = styled.h3`
+  color: #8a8e93;
+  font-size: 36px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 120%; /* 43.2px */
+
+  padding: 48px;
+  padding-top: 0;
+`;
+
+const Content = (
+  <>
+    <div className="w-100 d-flex position-relative align-items-center">
+      <div style={{ padding: 48, width: "60%" }}>
+        <DescriptionHeader>
+          Communities are the lifeblood of /dev/hub
+        </DescriptionHeader>
+        <Description>
+          We believe that communities are the foundation of a decentralized
+          ecosystem. Explore and engage with our diverse range of communities
+          today.
+        </Description>
+      </div>
+      <div
+        className="position-absolute top-25 end-0"
+        style={{ width: "50%", height: "65%" }}
+      >
+        <img
+          src={imageSource}
+          style={{
+            width: "100%",
+            height: "90%",
+            objectFit: "cover",
+            clipPath: "polygon(15% 0, 100% 0%, 100% 100%, 0% 100%)",
+            objectPosition: "center top",
+          }}
+        />
+      </div>
+    </div>
+    <Subheading>Featured Communities</Subheading>
+    <div
+      style={{ padding: 48, paddingTop: 0 }}
+      className="position-relative d-flex flex-row gap-3 w-100 align-items-center justify-content-center"
+    >
+      {Cards.slice(state.startIndex, state.endIndex + 1).map((card, idx) => (
+        <Card
+          title={card.title}
+          description={card.description}
+          href={card.href}
+          key={`project-card-${idx}`}
+        />
+      ))}
+      <ForwardButton
+        className="position-absolute end-0"
+        onClick={handleForward}
+      >
+        <svg
+          xmlns="http://www.w3.org/2000/svg"
+          width="48"
+          height="49"
+          viewBox="0 0 48 49"
+          fill="none"
+        >
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M32.9999 24.5L17.9999 39.5L15.8999 37.4L28.7999 24.5L15.8999 11.6L17.9999 9.5L32.9999 24.5Z"
+            fill="#00EC97"
+          />
+        </svg>
+      </ForwardButton>
+    </div>
+    <div style={{ padding: 48, paddingTop: 0 }}>
+      <CTA href="#">Explore all communities →</CTA>
+    </div>
+  </>
+);
+
+return (
+  <Widget
+    src="devhub.testnet/widget/home.components.Section.HomeSection"
```

Comment:
Replace with "${REPL_DEVHUB}/widget/home.components.Section.HomeSection"

Use this pattern for all these so it injects properly according to the network